src/components/Form/Form.js

```diff
@@ -0,0 +1,155 @@
+import React, {Component} from "react";
+import styled from "styled-components";
+import {COLOR_CLAVE_SKIN} from "../../colors";
+import "./error.css"
+
+class Form extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            navn: '',
+            Epost: '',
+            Melding: '',
+            nameValid: false,
+            emailValid: false,
+            messageValid: false,
+            formValid: false,
+            namePlaceholder: 'Navn',
+            emailPlaceholder: 'Epost',
+            messagePlaceholder: 'Melding',
+            nameIsChanged: false,
+            emailIsChanged: false,
+            messageIsChanged: false
+        }
+    }
```

Comment:
Ser du har skrevet ditt eget form rammeverk. Det er ikke så dumt det, men dersom du ender opp med mye corner cases og feil anbefaler jeg å heller bruke et bibliotek for dette som f.eks. https://github.com/jaredpalmer/formik